src/net.cpp

```diff
@@ -190,8 +190,8 @@ CAddress GetLocalAddress(const CNetAddr *paddrPeer, ServiceFlags nLocalServices)
 static int GetnScore(const CService& addr)
 {
     LOCK(cs_mapLocalHost);
-    if (mapLocalHost.count(addr) == 0) return 0;
-    return mapLocalHost[addr].nScore;
+    const auto it = mapLocalHost.find(addr);
+    return (it != mapLocalHost.end()) ? it->second.nScore : 0;
```

Comment:
Obviously doesn't matter here, but if there is an `else` to the `if`, I prefer to not invert the condition:


```cpp
     return it == mapLocalHost.end() ? 0 : it->second.nScore;